ruby-challenges.rb

```diff
@@ -19,17 +28,66 @@
 nums_array2 = [25, 17, 47, 11]
 # Expected output: 100
 
+def some_method (array)
+  array.sum do |value|
+    value
+  end
+end
+
+p some_method(nums_array1)
+p some_method(nums_array2)
```

Comment:
The `.sum` method is pretty cool. It will just add the contents of an array without needing a block.